### ISSUE TITLE
Check for Accessibility

### ISSUE THREAD

**Issue by podfeet** [podfeet/elapsed-time-adder#3]

**Comment by podfeet** [comment]
* values for total HMS are not accessible
* HMS headings aren't accessible
* minus and plus should say add row and subtract row

**Comment by podfeet** [comment]
After telling it to make it accessible, the +/- say add row and subtract row and explains in good words what will happen. However, to add a row has a different meaning in the interface. This should say Add time and subtract time

"Toggles whether the row is added or subtracted from the total"
